pkgs/applications/video/obs-studio/plugins/wlrobs.nix

```diff
@@ -5,14 +5,14 @@
 
 stdenv.mkDerivation {
   pname = "wlrobs";
-  version = "unstable-2022-10-06";
+  version = "unstable-2023-08-23";
 
   src = fetchFromSourcehut {
     vc = "hg";
     owner = "~scoopta";
     repo = "wlrobs";
-    rev = "78be323b25e1365f5c8f9dcba6938063ca10f71f";
-    sha256 = "sha256-/VemJkk695BdSDsODmYIPdhPwggzIhBi/0m6P+AYfx0=";
+    rev =  "f72d5cb3cbbd";
+    sha256 = "1nf8pg45swjwacvdaraa6l7c24i95pqds3rbradllj8jgxvk88w6";
```

Comment:
Could you use the full revision here, `f72d5cb3cbbd3983ae6cfd86cb1940be7372681c`? It's also probably a good idea to use an SRI-formatted hash, i.e. `hash = "sha256-hiM0d38SSUqbyisP3fAtKRLBDjVKZdU2U1xyXci7yNk="`, as those are becoming the default. You can use `nix to-sri sha256:<hash>` to convert them, and if you're using an older version of Nix, applying https://github.com/NixOS/nix/commit/63c5c91cc053cbc1fcb8d3fe71c41142c9f51bfa will give you SRI hashes in the hash mismatch warning. Otherwise, looks good! I've tested this and it seems to work just fine.

Reply:
How'd you get the full revision? I just used `nix-prefetch-hg` and it gave me the shorthash.

Reply:
It's in the URL of the commit: https://hg.sr.ht/~scoopta/wlrobs/rev/f72d5cb3cbbd3983ae6cfd86cb1940be7372681c

I don't know about how nix-prefetch-hg does things, but it truncating commit IDs sounds like something worth raising a ticket about.